<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="fileNameLabel">
     <property name="text">
      <string>&amp;File name:</string>
     </property>
     <property name="buddy">
      <cstring>fileButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="fileLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>./plot.png</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="unitScaleLabel">
       <property name="text">
        <string>&amp;Unit scale:</string>
       </property>
       <property name="buddy">
        <cstring>unitScaleEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExpressionLineEdit" name="unitScaleEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="pxLabel">
       <property name="text">
        <string>px</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="plotRangeLabel">
     <property name="text">
      <string>Plot range:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>&amp;x:</string>
       </property>
       <property name="buddy">
        <cstring>xMinEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="xFromLabel">
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExpressionLineEdit" name="xMinEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="xToLabel">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExpressionLineEdit" name="xMaxEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>&amp;y:</string>
       </property>
       <property name="buddy">
        <cstring>yMinEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yFromLabel">
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExpressionLineEdit" name="yMinEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExpressionLineEdit" name="yMaxEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>Image size: %1 x %2 [px]</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpressionLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../src/customcontrols.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fileButton</tabstop>
  <tabstop>unitScaleEdit</tabstop>
  <tabstop>xMinEdit</tabstop>
  <tabstop>xMaxEdit</tabstop>
  <tabstop>yMinEdit</tabstop>
  <tabstop>yMaxEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
